doc/Security.xml

```diff
@@ -3964,6 +3964,10 @@
                   <para>Indicates the maximum number of certificate paths that the device is able to store simultaneously.</para>
                 </entry>
               </row>
+              <row>
+                <entry>SetCertPath</entry>
+                <entry>Indicates support for modifying an existing certification path.</entry>
```

Comment:
Indicates support for modifying an existing certification path **and validation policy** as same capability is used to cover multiple APIs
  - SetCertificationPath
  - SetCertPathValidationPolicy

Refer to the "KeyStore capabilities" table reference 
- SetCertPath
  - The device supports modification of **certificate path and certificate validation path.**